### PR TITLE
Refresh editor's choice app list

### PR DESCRIPTION
**Pull request by bertob**
As discussed in https://github.com/flathub/linux-store-frontend/issues/220, here are some suggestions for a first refresh to the editor's choice list. Ideally we'd do something similar every few weeks so people always have something new to discover.

- Remove some of the very well-known apps that are also in Popular
- Remove Feedreader since it's EOL (it's been succeeded by [Newsflash](https://gitlab.com/news-flash/news_flash_gtk))
- Add some less well-known apps for people to discover

### CODE REVIEW

**Comment by bertob** [comment]
Can we merge this, or is there a blocker of some kind here?

File: scripts/update-editors-choice-apps-inputlist [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 org.openshot.OpenShot
 org.kde.okular
 im.riot.Riot
 net.scribus.Scribus
-org.gnome.PasswordSafe
\ No newline at end of file
+org.gnome.PasswordSafe
+org.gnome.design.Contrast
+com.uploadedlobster.peek
+com.vscodium.codium
```
Comment (TingPing) [inline scripts/update-editors-choice-apps-inputlist:22]: I think this is a terrible addition, it poorly works with a sandbox.

File: scripts/update-editors-choice-apps-inputlist [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
 com.slack.Slack
 org.inkscape.Inkscape
 org.blender.Blender
 org.darktable.Darktable
-org.videolan.VLC
 org.gabmus.hydrapaper
-org.filezillaproject.Filezilla
 org.kde.krita
-org.gimp.GIMP
-com.spotify.Client
```
Comment (TingPing) [inline scripts/update-editors-choice-apps-inputlist:15]: I think a few of these are terrible removals also. This is a very popular and well working app.

File: scripts/update-editors-choice-apps-inputlist (hunk 1 above)
Comment (bertob) [reply scripts/update-editors-choice-apps-inputlist:22]: Which of the three s a terrible addition? Feel free to remove it :)

File: scripts/update-editors-choice-apps-inputlist (hunk 2 above)
Comment (bertob) [reply scripts/update-editors-choice-apps-inputlist:15]: That's precisely why they don't need to be in editor's choice all the time. People already know these apps, and they're likely to be in the "popular" list as well.

The idea is that we'd update editor's choice regularly, and feature popular apps like GIMP sometimes, but not permanently.

Comment (TingPing) [reply scripts/update-editors-choice-apps-inputlist:15]: That does actually make sense.

File: scripts/update-editors-choice-apps-inputlist (hunk 1 above)
Comment (TingPing) [reply scripts/update-editors-choice-apps-inputlist]: Just vscode, done.

**Comment by bertob** [comment]
Any other blockers?

**Comment by jgarciao** [comment]
The blocker was me that didn't find the time to review it and merge it. Thanks for your contribution!

**Pull request merged**